translations/town.tex/pl.po

```diff
@@ -80,7 +80,21 @@ msgid ""
 "Some Building Tiles are double-sided, and may later be upgraded and flipped to represent two different buildings at the same time. Such upgrades must be \\textbf{Built in order}.\\par\n"
 "\n"
 msgstr ""
-
+"\\hypertarget{Miasto}{Każda}\\index{Miasto} frakcja ma własne miasto, które znajduje się na środku swojego kafla początkowego.\n"
+"Miasto jest twoją najważniejszą lokacją - wiele scenariuszy może się \\hyperlink{End}{zakończyć}, jeśli \\hyperlink{Categories}{oflaguje} je bohater wroga.\\par\n"
+"Stan twojego miasta i frakcji są ukazane na planszetce miasta.\n"
+"Pokazane są na niej wybudowane budynki, koszty budowy dostępnych budynków, przychód zasobów i stan żetonów Akcji.\\par\n"
+"Każda frakcja może wybudować następujące budynki\\index{Buildings}  w swoim mieście:\n"
+"\\begin{itemize}\n"
+"  \\item \\textbf{Ratusz} – W rundzie Zasobów, zyskujeszW rundzie Zasobów, zyskujesz przychód albo korzystasz z unikatowej zdolności swojej frakcji.\n"
```

Comment:
tu trochę odbiegamy od oryginału, który w tym akurat miejscu nie mówi, że przychód dostajemy w rundzie zasobów. czy takie przypadki załatwiamy przez issues?

 
```suggestion
"  \\item \\textbf{Ratusz} – W rundzie Zasobów, zyskujesz przychód albo korzystasz z unikatowej zdolności swojej Frakcji.\n"
```

Reply:
Ja bym zostawił. To _de facto_ dodatkowy przychód. Jest jeszcze nadmiarowy przecinek.